libraries/provision/install_couchbase_server.py

```diff
@@ -139,12 +142,13 @@ def install_couchbase_server(couchbase_server_config):
             "couchbase_server_package_name": server_package_name
         }
     )
-    assert status == 0, "Failed to install Couchbase Server"
+    if status != 0:
+        raise ProvisioningError("Failed to install Couchbase Server")
```

Comment:
At some point, not necessarily this PR, we should add a `must_run_ansible_playbook(.., errmsg="Failed to install Couchbase server")` which would raise a  inste ProvisioningError inside the method